website/docs/docs/build/snapshots.md

```diff
@@ -132,9 +165,40 @@ select * from {{ source('jaffle_shop', 'orders') }}
 
 6. Run the `dbt snapshot` [command](/reference/commands/snapshot) — for our example a new table will be created at `analytics.snapshots.orders_snapshot`. You can change the `target_database` configuration, the `target_schema` configuration and the name of the snapshot (as defined in `{% snapshot .. %}`) will change how dbt names this table.
 
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+<File name='snapshots/orders_snapshot.sql'>
+
+```sql
+{% snapshot orders_snapshot %}
+
+{{
+    config(
+      database='analytics',
+      schema='snapshots',
```

Comment:
Do we want to leave this in every example or remove them to stress that the fields are optional? Is leaving them undefined a best practice?

Reply:
I think let's leave `schema` as "best practice", but I think we can remove `database`